mobile/lib/screens/web_view_page.dart

```diff
@@ -4,7 +4,9 @@ import 'package:app/widgets/widgets.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:webview_flutter/webview_flutter.dart';
+// ignore: depend_on_referenced_packages, required by package
```

Comment:
not sure if we need these comments, do we? They do not seem to be important? Thoughts? @NoahNsimbe

Reply:
These comments prevent dart code metrics from raising errors like [depend_on_referenced_packages](https://dart-lang.github.io/linter/lints/depend_on_referenced_packages.html)